api/python/cellxgene_census/src/cellxgene_census/_get_anndata.py

```diff
@@ -22,6 +22,7 @@ def get_anndata(
     X_name: str = "raw",
     X_layers: Optional[Sequence[str]] = (),
     obsm_layers: Optional[Sequence[str]] = (),
```

Comment:
Two options:
1. Reuse `obsm_layers` (and `varm_layers`, that needs to be added) and modify the behavior so they retrieve the suitable embedding. This would effectively avoid introducing a breaking change
2. Rename the field, thus introducing a breaking change. 

If we don't plan to re-introduce a "mixed" scenario where we have both obsm objects in the census and external embeddings, I recommend 1.

Reply:
I don't like 1, as it is extremely confusing in a (likely future) world where we have both types.

I'd keep it super simple:  a param that says "add contributed embedding with algo name X".  You might also consider accepting the accession ID, as that resolves the case where there are non-unique algo names

Reply:
Yeah, I agree it could cause issues in the future. Let's go for the new param, unless @pablo-gar disagrees. I agree that accession id should be a valid choice. I'll add that to the docstrings.

Reply:
Added `add_obs_embeddings` and `add_var_embeddings`. Do we want to leave `obsm_layers` (and for completeness, add `varm_layers`)? The main concern is that right now, they would pull the "bogus" scvi embeddings so if we do, we should either make them no-op or add a warning. @pablo-gar any thoughts?

Reply:
I added a condition where `obsm_layers` and `add_obs_embedding` cannot share the same value. @atarashansky any idea on a better name for `add_obs_embedding` (from a user perspective)?